src/servicesHost.ts

```diff
@@ -14,15 +14,21 @@ import {
   Webpack
 } from './interfaces';
 
+export interface CachedServicesHost {
```

Comment:
Perhaps call this `ServiceHostWhichMayBeCacheable`?  It's super verbose but it's also very descriptive. Should make it clear to anyone hacking on the code.  I don't mind whether it's a `type` or an `interface`.

Reply:
Ok, what's about `servicesHost` property? Leave without changes?